<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPlainTextEdit" name="txtDocument">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>60</y>
      <width>291</width>
      <height>191</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Courier New</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="plainText">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="actionSave"/>
    <addaction name="actionSaveAs"/>
    <addaction name="separator"/>
    <addaction name="actionChangePassword"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menu_Edit">
    <property name="title">
     <string>&amp;Edit</string>
    </property>
    <addaction name="actionUndo"/>
    <addaction name="actionRedo"/>
    <addaction name="separator"/>
    <addaction name="actionCut"/>
    <addaction name="actionCopy"/>
    <addaction name="actionPaste"/>
    <addaction name="actionDelete"/>
    <addaction name="separator"/>
    <addaction name="actionSelectAll"/>
    <addaction name="separator"/>
    <addaction name="actionFind"/>
    <addaction name="actionFindNext"/>
    <addaction name="actionReplace"/>
    <addaction name="separator"/>
    <addaction name="actionInsertDateTime"/>
   </widget>
   <widget class="QMenu" name="menu_Format">
    <property name="title">
     <string>F&amp;ormat</string>
    </property>
    <addaction name="actionWordWrap"/>
    <addaction name="separator"/>
    <addaction name="actionFont"/>
   </widget>
   <widget class="QMenu" name="menu_Help">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menu_File"/>
   <addaction name="menu_Edit"/>
   <addaction name="menu_Format"/>
   <addaction name="menu_Help"/>
  </widget>
  <widget class="QToolBar" name="findBar">
   <property name="contextMenuPolicy">
    <enum>Qt::NoContextMenu</enum>
   </property>
   <property name="windowTitle">
    <string>Find</string>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="allowedAreas">
    <set>Qt::BottomToolBarArea|Qt::TopToolBarArea</set>
   </property>
   <property name="floatable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <action name="actionNew">
   <property name="text">
    <string>&amp;New</string>
   </property>
   <property name="toolTip">
    <string>Create a new secure file</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>&amp;Save</string>
   </property>
   <property name="toolTip">
    <string>Save your work</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionSaveAs">
   <property name="text">
    <string>Save &amp;As</string>
   </property>
   <property name="toolTip">
    <string>Save the current document somewhere else</string>
   </property>
  </action>
  <action name="actionChangePassword">
   <property name="text">
    <string>&amp;Change Password...</string>
   </property>
   <property name="toolTip">
    <string>Change your current password</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>E&amp;xit</string>
   </property>
   <property name="toolTip">
    <string>Quit this program</string>
   </property>
  </action>
  <action name="actionUndo">
   <property name="text">
    <string>&amp;Undo</string>
   </property>
   <property name="toolTip">
    <string>Undo the last action</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Z</string>
   </property>
  </action>
  <action name="actionRedo">
   <property name="text">
    <string>&amp;Redo</string>
   </property>
   <property name="toolTip">
    <string>Redo the last action</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Y</string>
   </property>
  </action>
  <action name="actionCut">
   <property name="text">
    <string>C&amp;ut</string>
   </property>
   <property name="toolTip">
    <string>Copy the selection and then delete it</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+X</string>
   </property>
  </action>
  <action name="actionCopy">
   <property name="text">
    <string>&amp;Copy</string>
   </property>
   <property name="toolTip">
    <string>Copy the selection to the clipboard</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="actionPaste">
   <property name="text">
    <string>&amp;Paste</string>
   </property>
   <property name="toolTip">
    <string>Paste the clipboard contents into the document</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+V</string>
   </property>
  </action>
  <action name="actionDelete">
   <property name="text">
    <string>De&amp;lete</string>
   </property>
   <property name="shortcut">
    <string>Del</string>
   </property>
  </action>
  <action name="actionSelectAll">
   <property name="text">
    <string>Select &amp;All</string>
   </property>
   <property name="toolTip">
    <string>Select the entire document</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+A</string>
   </property>
  </action>
  <action name="actionFind">
   <property name="text">
    <string>&amp;Find</string>
   </property>
   <property name="toolTip">
    <string>Search for text in the document</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+F</string>
   </property>
  </action>
  <action name="actionFindNext">
   <property name="text">
    <string>Find &amp;Next</string>
   </property>
   <property name="toolTip">
    <string>Find the next occurance of the last thing you searched for</string>
   </property>
   <property name="shortcut">
    <string>F3</string>
   </property>
  </action>
  <action name="actionReplace">
   <property name="text">
    <string>&amp;Replace</string>
   </property>
   <property name="toolTip">
    <string>Find and replace text</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+H</string>
   </property>
  </action>
  <action name="actionInsertDateTime">
   <property name="text">
    <string>Insert &amp;Date/Time</string>
   </property>
   <property name="toolTip">
    <string>Insert the current date and time into the document</string>
   </property>
   <property name="shortcut">
    <string>F5</string>
   </property>
  </action>
  <action name="actionWordWrap">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;Word Wrap</string>
   </property>
   <property name="toolTip">
    <string>Toggle whether long lines are wrapped</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>&amp;About...</string>
   </property>
   <property name="toolTip">
    <string>Learn about this program</string>
   </property>
  </action>
  <action name="actionFont">
   <property name="text">
    <string>&amp;Font...</string>
   </property>
   <property name="toolTip">
    <string>Change the font of the editor</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
